src/libexpr/primops/context.cc

```diff
@@ -119,7 +136,26 @@ static void prim_getContext(EvalState & state, const PosIdx pos, Value * * args,
     v.mkAttrs(attrs);
 }
 
-static RegisterPrimOp primop_getContext("__getContext", 1, prim_getContext);
+static RegisterPrimOp primop_getContext({
+    .name = "__getContext",
+    .args = {"s"},
+    .doc = R"(
+      Return the context associated with string *s*.
+
+      For example,
+
+      ```nix
+      builtins.getContext "X ${derivation { name = "a"; builder = "b"; system = "c"; }}"
```

Comment:
```suggestion
      builtins.getContext "${derivation { name = "a"; builder = "b"; system = "c"; }}"
```

The additional string contents are arbitrary and irrelevant for the example.